### PR TITLE
dont use django-jsonfield

### PR DESCRIPTION
**Pull request by czpython**
django-jsonfield and jsonfield both have the same package name and would cause bad conflicts when both are present in the python path.

I've moved the field logic from django-jsonfield into the `AttributesField`

### CODE REVIEW

File: djangocms_attributes_field/fields.py [hunk 1]
```diff
@@ -22,16 +21,31 @@
...
+
+    def validate(self, value):
+        # This is required in older django versions.
+        if value in self.empty_values and self.required:
+            raise forms.ValidationError(self.error_messages['required'], code='required')
 
-class AttributesField(jsonfield.JSONField):
+
+class AttributesField(models.Field):
     """
```
Comment (jmelett) [inline djangocms_attributes_field/fields.py]: Might remove this?

Comment (czpython) [inline djangocms_attributes_field/fields.py]: remove what? The base class is the only thing that changed there

Comment (jmelett) [inline djangocms_attributes_field/fields.py]: Sorry, meant the lines below, which mention that it's a sub-class of JSONField (which is not correct anymore)

**Comment by jmelett** [comment]
LGTM

**Pull request merged**